src/client.ts

```diff
@@ -4632,6 +4696,7 @@ export class MatrixClient extends TypedEventEmitter<EmittedEvents, ClientEventHa
         }
 
         // set up re-emitter for this new event - this is normally the job of EventMapper but we don't use it here
+        // TODO: skip for futures?
```

Comment:
any particular reason we don't do this already?

Reply:
Just a lack of testing.  I'll skip it now & will check that it doesn't break anything.